<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMatchingForm</class>
 <widget class="QFrame" name="CMatchingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matching form</string>
  </property>
  <layout class="QVBoxLayout" name="vl_MatchingForm">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="pb_ResetAll">
     <property name="text">
      <string>reset (all)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_Algorithm">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>Algorithm</string>
     </property>
     <layout class="QGridLayout" name="gl_Algorithm">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rb_ScoreOnly">
        <property name="text">
         <string>score only</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rb_Reduction">
        <property name="text">
         <string>reduction</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rb_ScoreAndReduction">
        <property name="text">
         <string>red. + score based</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pb_ResetAlgorithm">
        <property name="text">
         <string>reset (by algorithm)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_MatchingScript">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Matching script</string>
     </property>
     <layout class="QGridLayout" name="gl_MatchingScript" columnstretch="0,1,0,0">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_MsMatching">
        <property name="text">
         <string>Matching</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pb_MsMatching">
        <property name="text">
         <string>[...]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_MsReverseLookup">
        <property name="placeholderText">
         <string>reverse lookup matching script file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_MsMatching">
        <property name="placeholderText">
         <string>matching stage matching script file</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pb_MsReverseLookup">
        <property name="text">
         <string>[...]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_MsReverseLookup">
        <property name="text">
         <string>Lookup</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="cb_MsReverseLookup">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="cb_MsMatching">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_ReverseLookup">
     <property name="title">
      <string>Reverse lookup</string>
     </property>
     <layout class="QGridLayout" name="gl_ReverseLookup">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cb_ReverseUseModelString">
        <property name="text">
         <string>use model string</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cb_ReverseUseSwiftLiveryIds">
        <property name="text">
         <string>use swift livery ids</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_PickStrategy">
     <property name="title">
      <string>Similar results (strategy)</string>
     </property>
     <layout class="QHBoxLayout" name="hl_PickStrategy">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_PickFirst">
        <property name="text">
         <string>first</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_PickRandom">
        <property name="text">
         <string>random</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_PickByOrder">
        <property name="text">
         <string>higher rank first</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_Reduction">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>165</height>
      </size>
     </property>
     <property name="title">
      <string>Reduction</string>
     </property>
     <layout class="QGridLayout" name="gl_Mode">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rb_AirlineGroupNo">
        <property name="text">
         <string>no airline group</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_AirlineGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rb_ByIcaoDataAirline1st">
        <property name="text">
         <string>by ICAO: airline first</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_ICAOfirst</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rb_ByIcaoDataAircraft1st">
        <property name="text">
         <string>by ICAO: aircraft first</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_ICAOfirst</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="cb_ByFamily">
        <property name="text">
         <string>family</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="cb_ByLivery">
        <property name="text">
         <string> livery</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="cb_ByCivilian">
        <property name="toolTip">
         <string>inbound civilian try to match civilian</string>
        </property>
        <property name="text">
         <string>civilian</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="Line" name="line_Reduction">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QCheckBox" name="cb_ByVtol">
        <property name="toolTip">
         <string>Vertical Takeoff or Landing</string>
        </property>
        <property name="text">
         <string>VTOL (helicopters and such)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="cb_ByModelString">
        <property name="text">
         <string>model string</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="cb_ByCombinedCode">
        <property name="text">
         <string>combined code</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="rb_AirlineGroupIfNoAirline">
        <property name="text">
         <string>group if no airline</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_AirlineGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="cb_ByMilitary">
        <property name="toolTip">
         <string>inbound military try to find military</string>
        </property>
        <property name="text">
         <string>military</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="rb_AirlineGroupAsAirline">
        <property name="text">
         <string>group=airline</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">bg_AirlineGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_Categories">
     <property name="title">
      <string>Matching by categories</string>
     </property>
     <layout class="QGridLayout" name="gl_MacthingCategories">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cb_CategoryMilitaryAircraft">
        <property name="text">
         <string>military aircraft</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cb_CategoryGlider">
        <property name="text">
         <string>glider</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="cb_CategorySmallAircraft">
        <property name="text">
         <string>small aircraft</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_FailureHandling">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Failure handling and verification</string>
     </property>
     <layout class="QGridLayout" name="gl_FailureHandling">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cb_ModelSetRemoveFailed">
        <property name="toolTip">
         <string>temp.remove failed from model set</string>
        </property>
        <property name="text">
         <string>temp.remove failed</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cb_ModelSetVerification">
        <property name="text">
         <string>verify set on startup</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cb_ModelFailedFailover">
        <property name="text">
         <string>use other model if failed</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cb_ModelSetVerificationOnlyErrorWarning">
        <property name="text">
         <string>dialog only if warnings/error</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbExclusion">
     <property name="title">
      <string>Exclusion</string>
     </property>
     <layout class="QHBoxLayout" name="hl_Exclusion">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cb_ExclNoExcludedModels">
        <property name="text">
         <string>excluded models</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_ExclNoDbData">
        <property name="text">
         <string>exclude if no DB models</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_Scoring">
     <property name="title">
      <string>Scoring</string>
     </property>
     <layout class="QHBoxLayout" name="hl_Scoring">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cb_ScorePreferColorLiveries">
        <property name="text">
         <string>prefer color liveries</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_ScoreIgnoreZeros">
        <property name="text">
         <string>ignore 0 (zero) scores</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pb_ResetAll</tabstop>
  <tabstop>rb_Reduction</tabstop>
  <tabstop>rb_ScoreOnly</tabstop>
  <tabstop>rb_ScoreAndReduction</tabstop>
  <tabstop>pb_ResetAlgorithm</tabstop>
  <tabstop>le_MsReverseLookup</tabstop>
  <tabstop>cb_MsReverseLookup</tabstop>
  <tabstop>pb_MsReverseLookup</tabstop>
  <tabstop>le_MsMatching</tabstop>
  <tabstop>cb_MsMatching</tabstop>
  <tabstop>pb_MsMatching</tabstop>
  <tabstop>cb_ReverseUseModelString</tabstop>
  <tabstop>cb_ReverseUseSwiftLiveryIds</tabstop>
  <tabstop>rb_PickFirst</tabstop>
  <tabstop>rb_PickRandom</tabstop>
  <tabstop>rb_PickByOrder</tabstop>
  <tabstop>rb_ByIcaoDataAircraft1st</tabstop>
  <tabstop>rb_ByIcaoDataAirline1st</tabstop>
  <tabstop>rb_AirlineGroupNo</tabstop>
  <tabstop>rb_AirlineGroupIfNoAirline</tabstop>
  <tabstop>rb_AirlineGroupAsAirline</tabstop>
  <tabstop>cb_ByModelString</tabstop>
  <tabstop>cb_ByLivery</tabstop>
  <tabstop>cb_ByFamily</tabstop>
  <tabstop>cb_ByCombinedCode</tabstop>
  <tabstop>cb_ByVtol</tabstop>
  <tabstop>cb_ByMilitary</tabstop>
  <tabstop>cb_ByCivilian</tabstop>
  <tabstop>cb_CategoryGlider</tabstop>
  <tabstop>cb_CategoryMilitaryAircraft</tabstop>
  <tabstop>cb_CategorySmallAircraft</tabstop>
  <tabstop>cb_ModelSetVerification</tabstop>
  <tabstop>cb_ModelSetVerificationOnlyErrorWarning</tabstop>
  <tabstop>cb_ModelSetRemoveFailed</tabstop>
  <tabstop>cb_ModelFailedFailover</tabstop>
  <tabstop>cb_ExclNoExcludedModels</tabstop>
  <tabstop>cb_ExclNoDbData</tabstop>
  <tabstop>cb_ScorePreferColorLiveries</tabstop>
  <tabstop>cb_ScoreIgnoreZeros</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bg_AirlineGroup"/>
  <buttongroup name="bg_ICAOfirst"/>
 </buttongroups>
</ui>
